libs/design/theme/theme.ts

```diff
@@ -75,7 +75,19 @@ export const theme: MantineThemeOverride = {
   primaryColor: 'indigo',
   defaultRadius: 'sm',
   components: {
+    DateInput: {
+      styles: {
+        label: {
+          color: colors.fgSubtle,
+        },
+      },
+    },
     TextInput: {
+      styles: {
+        label: {
+          color: colors.fgSubtle,
```

Comment:
Bro, u shouldn't write this style globally. cause it changes the `CreateFamily` and `CreateProject` input styles.